### ISSUE TITLE
Where to describe equipment and protocol used for machine observations

### ISSUE THREAD

**Issue by peterdesmet** [tdwg/dwc-for-biologging#10]
Original question by @sarahcd:

> Here I compiled the available sensor information (see the Movebank-format “reference data”) under samplingProtocol in the event and occurrence tables but that seemed a bit of a messy way to describe equipment, especially for cases where more sensor info is available and where detection limits or accuracy might be critical to interpreting measurements. For example, Movebank has a number of attributes used to describe [tags](http://movebank.org/node/2381#tags) and [tag deployments](http://movebank.org/node/2381) on animals. In my experience these are a pretty basic set of attributes that can cover most information commonly available for most animal tracking data. Additional DwC terms could also take advantage of more thorough vocabularies, e.g. the SensorML vocabularies the BODC maintains for marine sensor data.

**Comment by peterdesmet** [comment]
I agree with @albenson-usgs, that info could be added as facts about the deployment:

id | measurementType | measurementValue
--- | --- | ---
IDCoy_P3_1:deployment1 | attachment type | collar
IDCoy_P3_1:deployment1 | deployment end type | removal
IDCoy_P3_1:deployment1 | deployment id | 166760_Winter
IDCoy_P3_1:deployment1 | duty cycle | 5-min fixes
IDCoy_P3_1:deployment1 | manipulation type | none
IDCoy_P3_1:deployment1 | study site | Idaho National Engineering Laboratory
IDCoy_P3_1:deployment1 | tag manufacturer name | Lotek
IDCoy_P3_1:deployment1 | tag model | 
IDCoy_P3_1:deployment1 | tag readout method | tag-retrieval

I think all of these are related to the deployment and not to the capture/recapture. Ideally they would also have a measurementValueID that links out to a definition for each of these facts/settings.

**Comment by sarahcd** [comment]
Thanks @albenson-usgs and @peterdesmet. Suggested additions to the MOF table made for V3 of the Mahoney case study, including measurementValueID.
